docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -45,17 +45,17 @@ You can use Camunda 8 to orchestrate processes with human tasks of any complexit
 
 This guide introduces you to the basics of human task orchestration. You will create a simple process to decide on what to eat for dinner, and drive the process flow according to that decision.
 
-- Select **Self-Managed** to see an example using Camunda 8 Run and Desktop Modeler.
-- Select **SaaS** to see an example using Camunda 8 SaaS and Web Modeler.
+- Select **Desktop** to see an example using Camunda 8 Run and Desktop Modeler in a local development environment.
+- Select **Web** to see an example using Camunda 8 SaaS and Web Modeler.
```

Comment:
Web Modeler Self-Managed is excluded, but that's appropriate for this stage of the journey